### PR TITLE
chore: Fix layout of y axis labels

### LINKED ISSUE #27705: [Bug]: The y labels on a basic chart are overlapping with each other
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Screenshot 2023-09-29 at 6 50 50 AM

The gap between the labels and the y axis name should be more.

### Steps To Reproduce

DnD a Chart Widget.
Notice the gap isn't too much.

### Public Sample App

_No response_

### Environment

Production

### Issue video log

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by rajatagrawal**
Fixes #27705

### CODE REVIEW

**Review by sbalaji1192 — APPROVED** [review]

**Comment by laveena-en** [comment]
@rajatagrawal Overlap is still seen on resizing the widget. Attaching image below for reference: 
image

**Comment by rajatagrawal** [comment]
@laveena-en 

Can you please share the x,y data that is used for reproducing the bug ?

**Comment by laveena-en** [comment]
@rajatagrawal This is seen with the default data itself. No changes were made to the config 
image

**Comment by laveena-en** [comment]
Verified. Changes look good to me!

File: app/client/src/widgets/ChartWidget/component/helpers.ts [hunk 1]
```diff
@@ -209,3 +210,7 @@ export const isCustomFusionChart = (type: ChartType) => {
 export const isCustomEChart = (type: ChartType) => {
   return type == "CUSTOM_ECHART";
 };
+
+export const numberInLocale = (label: number | string) => {
+  return label.toLocaleString();
```
Comment (sbalaji1192) [inline app/client/src/widgets/ChartWidget/component/helpers.ts:215]: no need to create a helper that just calls a native function.

**Review by sbalaji1192 — APPROVED** [review]

**Pull request merged**